components/layout/Header.js

```diff
@@ -6,39 +6,45 @@ import Cart from '../cart/Cart';
 
 const Header = () => (
   <StyledHeader>
-    <div className="bar">
-      <StyledLogo>
-        <Link href="/">hello</Link>
-      </StyledLogo>
-      <Nav />
-    </div>
+    {/* <div className="bar"> */}
+    <StyledLogo>
+      <Link href="/">hello</Link>
+    </StyledLogo>
+    <Nav />
+    {/* </div> */}
     <Cart />
   </StyledHeader>
 );
 
+const StyledHeader = styled.header`
+  padding: 0 2rem;
+  margin-bottom: 2rem;
+  display: grid;
```

Comment:
I think this is over engineered. You can do this probably by using one of the two (flex or grid). Don't use both.
I think `flex` would be the correct one

Reply:
Oh, do you mean don't switch to flex when the screen gets small? Or, can you be more specific please (if you have time) The orgs I'm looking at for grid inspiration are all using grid and flex together in a complimentary way. I'm trying to learn from that, but this is the first time I'm trying to implement that 😕

Reply:
To clarify: Use either flex or grid on all screen sizes; not the two.
Flex and grid are both very powerful and very good for responsive design.

The only reason I see fit to use flex and grid together is for example when you nest a flex item inside a grid item (or visa-versa), but I don't see the need to toggle between the two.
Try to fix it by using one of the two on all screen sizes (and like I said; I think it could be done with only `flex`).

Let me know if you don't know how and we can look at it together 👍

Reply:
I did it on my lunch with grid, it was so easy. I want to practice using grid because all my other projects are with flexbox only. If you are into it, I have some grid related questions... But they are just for fun, so no pressure☺️

Reply:
I can definitely encourage practicing `grid` and other concepts you're not super comfortable with. But that shouldn't go in kost of making flexible, simple and clean code (not saying that yours is not any of this, but just making the statement).
I do think Flexbox is the better choice, but I'm not certain, as I'm seeing your application based on the code. Maybe if I would see it in the browser I would have a different opinion, so I leave it to you to asses the situation. As long as you use only one of the two. Not both.

And of course you can always ask me questions regarding `grid` and any other developer related topic.
Messsage me on Slack 😄